bakery/src/scripts/mathmltable2png.py

```diff
@@ -0,0 +1,188 @@
+from lxml import etree
+import os
+import io
+import sys
+import requests
+import base64
+import hashlib
+import struct
+
+DENSITY_DPI = 900
+IMG_SNIPPET = '<img src="{}" alt="{}" width="{}" height="{}" />'
+
+
+def force_math_namespace_only(doc):
+    # http://wiki.tei-c.org/index.php/Remove-Namespaces.xsl
+    xslt = u'''<xsl:stylesheet
+      version="1.0"
+      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+      xmlns="http://www.w3.org/1998/Math/MathML">
+    <xsl:output method="xml" indent="no"/>
+
+    <xsl:template match="/|comment()|processing-instruction()">
+        <xsl:copy>
+          <xsl:apply-templates/>
+        </xsl:copy>
+    </xsl:template>
+
+    <xsl:template match="*">
+        <xsl:element name="{local-name()}">
+          <xsl:apply-templates select="@*|node()"/>
+        </xsl:element>
+    </xsl:template>
+
+    <xsl:template match="@*">
+        <xsl:attribute name="{local-name()}">
+          <xsl:value-of select="."/>
+        </xsl:attribute>
+    </xsl:template>
+    </xsl:stylesheet>
+    '''
+    xslt_doc = etree.parse(io.StringIO(xslt))
+    transform = etree.XSLT(xslt_doc)
+    doc = transform(doc)
+    return doc
+
+
+def strip_mathjax_container(svg):
+    xslt = u'''<xsl:stylesheet
+      version="1.0"
+      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+      xmlns="http://www.w3.org/1998/Math/MathML">
+    <xsl:output method="xml" indent="no"/>
+
+    <xsl:template match="mjx-container">
+        <xsl:apply-templates/>
+    </xsl:template>
+
+    <xsl:template match="@*|node()">
+        <xsl:copy>
+            <xsl:apply-templates select="@*|node()"/>
+        </xsl:copy>
+    </xsl:template>
+
+    </xsl:stylesheet>
+    '''
+    xslt_doc = etree.parse(io.StringIO(xslt))
+    svg_xml = etree.parse(io.StringIO(svg))
+    transform = etree.XSLT(xslt_doc)
+    svg_xml = transform(svg_xml)
+    bytes_svg = etree.tostring(svg_xml, with_tail=False)
+    pure_svg = str(bytes_svg, 'utf-8')
+    return pure_svg
+
+
+def get_png_dimensions(png_bytes):
+    check = struct.unpack('>i', png_bytes[4:8])[0]
+    if check != 0x0d0a1a0a:
+        return 0, 0
+    width, height = struct.unpack('>ii', png_bytes[16:24])
+    return width, height
+
+
+def mathml2svg_jsonrpc(equation):
+    url = "http://localhost:3000/jsonrpc"
```

Comment:
Minor: This URL is defined twice here and in `svg2png_jsonrpc`. Maybe worth defining as a constant or refactoring a general shared function for making a request to the server?